cli/pkg/kctrl/cmd/package/release/helm_openapi_schema_gen_test.go

```diff
@@ -0,0 +1,220 @@
+// Copyright 2022 VMware, Inc.
+// SPDX-License-Identifier: Apache-2.0
+
+package release
```

Comment:
should be under _test suffix.

Reply:
if it is about filename, filename has test suffix`helm_openapi_schema_gen_test.go`